Mathlib/Control/Combinators.lean

```diff
@@ -68,17 +69,20 @@ def sequence {m : Type u → Type v} [Monad m] {α : Type u} : List (m α) → m
     return (h' :: t')
 
 /-- Executes a list of monadic actions in sequence, discarding the results. -/
+@[deprecated "Use `_root_.sequence` instead." (since := "2025-04-07")]
 def sequence' {m : Type → Type u} [Monad m] {α : Type} : List (m α) → m Unit
   | [] => return ()
   | h :: t => h *> sequence' t
 
 /-- Executes `t` if `b` is `true`, doing nothing otherwise.
 
 See also `when` and `whenM`. -/
+@[deprecated "No replacement in Mathlib." (since := "2025-04-07")]
 def whenb {m : Type → Type} [Monad m] (b : Bool) (t : m Unit) : m Unit :=
   _root_.cond b t (return ())
 
 /-- Executes `t` if `b` is `false`, doing nothing otherwise. -/
+@[deprecated "No replacement in Mathlib." (since := "2025-04-07")]
```

Comment:
This one has something of a replacement by `unless` in do-notation, which you could mention here?